src/routes/index.js

```diff
@@ -3,6 +3,6 @@ import apiRouter from './api';
 
 const index = express.Router();
 
-index.use('/api', apiRouter);
+const foo = (io = null) => index.use('/api', apiRouter(io));
 
-export default index;
+export default foo;
```

Comment:
Is there a reason why you used 'foo' a variable name? I think a name that helps understand what the we seek to achieve might be better